orca/orca-webhook/src/main/java/com/netflix/spinnaker/orca/webhook/tasks/WebhookResponseProcessor.java

```diff
@@ -137,19 +142,19 @@ private TaskResult processClientException(Exception e) {
               "Remote host resolution failure in webhook for pipeline %s to %s, will retry.",
               executionId, stageData.url);
       executionStatus = ExecutionStatus.RUNNING;
-    } else if (stageData.method == HttpMethod.GET
-        && (e instanceof SocketTimeoutException
-            || e.getCause() instanceof SocketTimeoutException)) {
+    } else if ((e instanceof SocketTimeoutException
+            || e.getCause() instanceof SocketTimeoutException)
+        && ((stageData.method == HttpMethod.GET) || safeToRetry(restTemplateData))) {
```

Comment:
nit: could move `stageData.method == HttpMethod.GET` into `safeToRetry` as a short circuit to simplify this if statement

Reply:
Yeah, I'd tempted to leave it like this.  I dunno, six of one, half a dozen of the other.

Reply:
Fine by me